storybook/stories/API/cartesian/Scatter.stories.tsx

```diff
@@ -0,0 +1,576 @@
+  onAnimationStart,
+  onAnimationStartCapture,
+  onAuxClick,
+  onAuxClickCapture,
+  onBeforeInput,
+  onBeforeInputCapture,
+  onBlur,
+  onBlurCapture,
+  onCanPlay,
+  onCanPlayCapture,
+  onCanPlayThrough,
+  onCanPlayThroughCapture,
+  onChange,
+  onChangeCapture,
+  onClick,
+  onClickCapture,
+  onCompositionEnd,
+  onCompositionEndCapture,
+  onCompositionStart,
+  onCompositionStartCapture,
+  onCompositionUpdate,
+  onCompositionUpdateCapture,
+  onContextMenu,
+  onContextMenuCapture,
+  onCopy,
+  onCopyCapture,
+  onCut,
+  onCutCapture,
+  onDoubleClick,
+  onDoubleClickCapture,
+  onDrag,
+  onDragCapture,
+  onDragEnd,
+  onDragEndCapture,
+  onDragEnter,
+  onDragEnterCapture,
+  onDragExit,
+  onDragExitCapture,
+  onDragLeave,
+  onDragLeaveCapture,
+  onDragOver,
+  onDragOverCapture,
+  onDragStart,
+  onDragStartCapture,
+  onDrop,
+  onDropCapture,
+  onDurationChange,
+  onDurationChangeCapture,
+  onEmptied,
+  onEmptiedCapture,
+  onEncrypted,
+  onEncryptedCapture,
+  onEnded,
+  onEndedCapture,
+  onError,
+  onErrorCapture,
+  onFocus,
+  onFocusCapture,
+  onGotPointerCapture,
+  onGotPointerCaptureCapture,
+  onInput,
+  onInputCapture,
+  onInvalid,
+  onInvalidCapture,
+  onKeyDown,
+  onKeyDownCapture,
+  onKeyPress,
+  onKeyPressCapture,
+  onKeyUp,
+  onKeyUpCapture,
+  onLoad,
+  onLoadCapture,
+  onLoadedData,
+  onLoadedDataCapture,
+  onLoadedMetadata,
+  onLoadedMetadataCapture,
+  onLoadStart,
+  onLoadStartCapture,
+  onLostPointerCapture,
+  onLostPointerCaptureCapture,
+  onMouseDown,
+  onMouseDownCapture,
+  onMouseEnter,
+  onMouseLeave,
+  onMouseMove,
+  onMouseMoveCapture,
+  onMouseOut,
+  onMouseOutCapture,
+  onMouseOver,
+  onMouseOverCapture,
+  onMouseUp,
+  onMouseUpCapture,
+  onPaste,
+  onPasteCapture,
+  onPause,
+  onPauseCapture,
+  onPlay,
+  onPlayCapture,
+  onPlaying,
+  onPlayingCapture,
+  onPointerCancel,
+  onPointerCancelCapture,
+  onPointerDown,
+  onPointerDownCapture,
+  onPointerEnter,
+  onPointerEnterCapture,
+  onPointerLeave,
+  onPointerLeaveCapture,
+  onPointerMove,
+  onPointerMoveCapture,
+  onPointerOut,
+  onPointerOutCapture,
+  onPointerOver,
+  onPointerOverCapture,
+  onPointerUp,
+  onPointerUpCapture,
+  onProgress,
+  onProgressCapture,
+  onRateChange,
+  onRateChangeCapture,
+  onReset,
+  onResetCapture,
+  onScroll,
+  onScrollCapture,
+  onSeeked,
+  onSeekedCapture,
+  onSeeking,
+  onSeekingCapture,
+  onSelect,
+  onSelectCapture,
+  onStalled,
+  onStalledCapture,
+  onSubmit,
+  onSubmitCapture,
+  onSuspend,
+  onSuspendCapture,
+  onTimeUpdate,
+  onTimeUpdateCapture,
+  onTouchCancel,
+  onTouchCancelCapture,
+  onTouchEnd,
+  onTouchEndCapture,
+  onTouchMove,
+  onTouchMoveCapture,
+  onTouchStart,
+  onTouchStartCapture,
+  onTransitionEnd,
+  onTransitionEndCapture,
+  onVolumeChange,
+  onVolumeChangeCapture,
+  onWaiting,
+  onWaitingCapture,
+  onWheel,
+  onWheelCapture,
+} from '../props/EventHandlers';
```

Comment:
We might be able to group these EventHandlers, in case they are always found together. So far it seems not all EventHandlers are available to all components, and I am not sure what logic this follows.